Tests/Functional/DependencyResolutionTests.swift

```diff
@@ -22,6 +22,12 @@ class DependencyResolutionTestCase: XCTestCase {
         }
     }
 
+    func testInternalExecAsDep() {
+        fixture(name: "DependencyResolution/Internal/InternalExecutableAsDependency") { prefix in
```

Comment:
Hmm I think it would be difficult to test this in a non functional way since this magic happens in Transmute which is all about scanning the directories.